### ISSUE TITLE
Perl dependencies in README.md 

### ISSUE THREAD

**Issue by osresearch** [openbmc/openbmc#3404]
Fedora required additional dependencies to be able to build OpenBMC:

```
sudo dnf install perl-Thread-Queue perl-bignum perl-Crypt-OpenSSL-Bignum
```

The first is detected before the build; the others not until task 200 or so.

**Comment by amboar** [comment]
Well that got noisy, should probably find a better way to view the rendered markdown. Anyway, resolved by https://gerrit.openbmc-project.xyz/c/openbmc/openbmc/+/20528 - sorry that this got overlooked.

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/c/openbmc/openbmc/+/20528> README: Document additional required Fedora dependencies
Resolves  #3404 Perl dependencies in README.md
